jgrapht-core/src/main/java/org/jgrapht/alg/spanning/PrimMinimumSpanningTree.java

```diff
@@ -17,21 +17,29 @@
  */
 package org.jgrapht.alg.spanning;
 
-import java.util.*;
+import org.jgrapht.Graph;
+import org.jgrapht.Graphs;
+import org.jgrapht.alg.interfaces.SpanningTreeAlgorithm;
+import org.jgrapht.util.FibonacciHeap;
+import org.jgrapht.util.FibonacciHeapNode;
 
-import org.jgrapht.*;
-import org.jgrapht.alg.interfaces.*;
+import java.lang.reflect.Array;
+import java.util.*;
 
 /**
  * An implementation of <a href="http://en.wikipedia.org/wiki/Prim's_algorithm"> Prim's
  * algorithm</a> that finds a minimum spanning tree/forest subject to connectivity of the supplied
  * weighted undirected graph. The algorithm was developed by Czech mathematician V. Jarník and later
  * independently by computer scientist Robert C. Prim and rediscovered by E. Dijkstra.
  *
+ * This implementation relies on a priority queue, and runs in $O(|E|log(|V|))$.
```

Comment:
This implementation relies on a Fibonacci Heap...

Reply:
Solved. I also changed the complexity to O(E + VlogV).